api/android/api/src/main/jni/nnstreamer-native.h

```diff
@@ -43,6 +43,13 @@
 #define CAST_TO_TYPE(l,type) (type)(jint)(l)
 #endif
 
+/**
+ * @brief NNStreamer package name.
+ */
+#define NNS_PKG "org/nnsuite/nnstreamer"
```

Comment:
BTW, are we still this organization, `nnsuite`?

Reply:
yes, in android (jcenter) we set the group name 'org.nnsuite'
    userOrg = 'nnsuite'
    repoName = 'nnstreamer'
    groupId = 'org.nnsuite'